Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs

```diff
@@ -328,7 +328,7 @@ private void UnMapSelectedElementRowsHandler(object sender, RoutedEventArgs e)
             }
             catch (Exception ex)
             {
-                Reporter.ToLog(eLogLevel.DEBUG, ex.Message);
+                Reporter.ToLog(eLogLevel.DEBUG, ex.ToString());
```

Comment:
change it to Reporter.ToLog(eLogLevel.DEBUG, "", ex);

Reply:
Done